waspc/src/Wasp/Project/Db/Dev/Postgres.hs

```diff
@@ -0,0 +1,35 @@
+-- | This module captures how Wasp runs a PostgreSQL dev database.
+module Wasp.Project.Db.Dev.Postgres
+  ( defaultDevUser,
+    makeDevDbName,
+    defaultDevPass,
+    defaultDevPort,
+    makeDevConnectionUrl,
+  )
+where
+
+import StrongPath (Abs, Dir, Path')
+import Wasp.Db.Postgres (makeConnectionUrl, postgresMaxDbNameLength)
+import Wasp.Project.Common (WaspProjectDir)
+import Wasp.Project.Db.Dev (makeDevDbUniqueId)
+
+defaultDevUser :: String
+defaultDevUser = "postgresWaspDevUser"
+
+defaultDevPass :: String
+defaultDevPass = "postgresWaspDevPass"
+
+makeDevDbName :: Path' Abs (Dir WaspProjectDir) -> String -> String
+makeDevDbName waspProjectDir appName =
+  -- We use makeDevDbUniqueId to construct a db name instead of a hardcoded value like "waspDevDb"
+  -- in order to avoid the situation where one Wasp app accidentally connects to a db that another
+  -- Wasp app has started. This way db name is unique for the specific Wasp app, and another Wasp app
```

Comment:
Is it possible to create multiple databases in an existing Docker container? I thought those env vars (`DATABASE_NAME` or something similar) you pass it on start were only used then and subsequent runs ignored them?

Reply:
No, not really! At least I don't think this is the idea how this Docker image should be used, nor do we do it.

Yes, they are used only once. Or to put it better, they are used only if initialized db is not already present on disk. And disk in this case is docker volume. And we have one docker volume per one wasp app.

When I say we want to avoid situation where one app connects to another one's db is following: imagine you have two Wasp apps, WA1 and WA2. In WA1 you run `wasp start db`. Then you go to WA2 and run `wasp start` there. If WA1 hosted that db on the same connection URL as WA2 is using to connect to dev db, then WA2 will happily connect to it and you will never know. Since connection url is unique in following: db name, user name, pass, port, then if all of those are the same, this can happen!
Which is why I make it so that db name is unique per app, so this can't happen.